### PR TITLE
Automation/6808-chanel-create-system-for-updating-names-on-release-tickets

### PR DESCRIPTION
**Pull request by IsraelleHub**
## Description of Change
Updating names on the the release ticket is a manual process right now

Investigate if we can auto-assign people to the release ticket based on the folks listed in a github group. We should be able to read the people in a certain group, then assign them to the release ticket instead of having to maintain a manual list inside the workflow like we currently do.

[release issue example](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/.github/ISSUE_TEMPLATE/release_ticket.md?plain=1#L6)
[release issue creation workflow](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/.github/workflows/release_branch_issue.yml#L34)
## Testing
Test was done by cloning the release_issue.md file and using a test workflow.
[Example of release ticket](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/10090) that the test workflow created and auto-assigned members of the [flagship-mobile-release-approvers](https://github.com/orgs/department-of-veterans-affairs/teams/flagship-mobile-release-approvers) using gh cli.
In addition, workflow update the assignee of release_ticket md file automatically whenever there is any change on the https://github.com/orgs/department-of-veterans-affairs/teams/flagship-mobile-release-approvers/members goup

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

**Comment by IsraelleHub** [comment]
This PR is ready for review. However I will still delete the clone release issue template md file as well as the test file once review finalize. We will also need to add the rest of the release issue template assignee on the flagship-mobile-release-approvers group once review finalized.

**Comment by timwright12** [comment]
> This PR is ready for review. However I will still delete the clone release issue template md file as well as the test file once review finalize. We will also need to add the rest of the release issue template assignee on the flagship-mobile-release-approvers group once review finalized.

@IsraelleHub is there a reason we can't do this stuff before merging code?

**Comment by IsraelleHub** [comment]
> > This PR is ready for review. However I will still delete the clone release issue template md file as well as the test file once review finalize. We will also need to add the rest of the release issue template assignee on the flagship-mobile-release-approvers group once review finalized.
> 
> @IsraelleHub is there a reason we can't do this stuff before merging code?

In case I need to run another test, if we add the rest of the assignee, it might be all of them. If this is not a problem, we can add them now.  
Also, I am leaving the the test workflow and the md file just in case I need to run more test based on the review.  But everything looks pretty good based on all the test I did so far!

File: .github/ISSUE_TEMPLATE/update_releaseissue.md
Comment (IsraelleHub) [reply .github/ISSUE_TEMPLATE/update_releaseissue.md:2]: Yes. this was just for test purposes

File: .github/workflows/release_branch_issue.yml
Comment (timwright12) [inline .github/workflows/release_branch_issue.yml:37]: Are we updating the ticket template itself and checking in that change instead of assigning it to the correct users after creation?

File: .github/workflows/release_branch_issue.yml
Comment (timwright12) [inline .github/workflows/release_branch_issue.yml:41]: Remove debugging before merging

File: .github/workflows/release_branch_issue.yml
Comment (timwright12) [inline .github/workflows/release_branch_issue.yml:23]: Is this PAT tied to your github account?

Comment (IsraelleHub) [reply .github/workflows/release_branch_issue.yml:23]: no. It's not!

File: .github/workflows/release_branch_issue.yml
Comment (IsraelleHub) [reply .github/workflows/release_branch_issue.yml:37]: Good question! Yes, based on the workflow steps, it appears that the ticket template file itself (the markdown file) is being updated with assignees and committed back to the repository. However, if the goal is to only assign the correct users to the created ticket without modifying and committing the template file. Please let me know

Comment (IsraelleHub) [reply .github/workflows/release_branch_issue.yml:37]: Also, if that's the case! should we remove the assignee line on the release_ticket md file?

Comment (timwright12) [reply .github/workflows/release_branch_issue.yml:37]: I think we should keep someone assigned to the ticket by default just in case something fails, but other than that I'm OK managing this by just assigning the ticket in CI to the group users

Comment (IsraelleHub) [reply .github/workflows/release_branch_issue.yml:37]: Ok. I will remove 'the update md file' line as well as the 'Commit and Push Assignees Update to Markdown' step. This way it will just auto assign user from the GitHub group to the release ticket. We will keep the assignee on the md file as precaution. Is this ok?

File: .github/workflows/release_branch_issue.yml
Comment (narin) [inline .github/workflows/release_branch_issue.yml:51]: Please add space after `#` if keeping this comment

File: .github/workflows/release_branch_issue.yml
Comment (narin) [inline .github/workflows/release_branch_issue.yml:55]: Remove these lines if they are for testing only

**Review by narin — APPROVED** [review]
Looks good. Some minor nitpicks but should be good to go otherwise.

**Review by narin — APPROVED** [review]

**Pull request merged**
